src/contexts/Validators/ValidatorEntries/index.tsx

```diff
@@ -239,10 +239,11 @@ export const ValidatorsProvider = ({
   // Subscribe to active session validators.
   const subscribeSessionValidators = async () => {
     if (!api || !isReady) return;
-    const unsub: AnyApi = await api.query.session.validators((v: AnyApi) => {
-      setSessionValidators(v.toHuman());
-      sessionUnsub.current = unsub;
-    });
+    const sessionValidatorsRaw: AnyApi =
+      await api.query.staking.validators.entries();
```

Comment:
I'm not sure if this should be really session validators or eras validators. Either way `staking.validators` is not the right storage :sweat_smile: 

This returns not only era's validators, but also as stated [here](https://github.com/paritytech/polkadot-sdk/blob/0e09ad448bce27fcd255370cc2827ea5d2cf3892/substrate/frame/staking/src/pallet/mod.rs#L408) "wannabe". I think we should use `erasStakersOverview` to get era's validators or `session.validators` to get session's validators. I'll try to figure out which one are we interested in :sweat_smile:

Reply:
I'm confused. On production `session.validators` is used, but I think `staking.erasStakersOverview` makes more sense. @Marcin-Radecki WDYT?

Reply:
Fixed
https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/pull/84/files#diff-7fe17c11d90b2086e1dd7f8d348e295c9a5ab9dd06dc2ce77d1b16fdf6b7b76cR243